client/components/tooltip/docs/example.jsx

```diff
@@ -2,47 +2,45 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
 import PureRenderMixin from 'react-pure-render/mixin';
 
 /**
  * Internal dependencies
  */
 import Tooltip from 'components/tooltip';
 
-const TooltipInstance = React.createClass( {
-	mixins: [ PureRenderMixin ],
-
-	componentWillMount() {
+class TooltipInstance extends Component {
+	constructor() {
+		super();
+		this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate.bind( this );
```

Comment:
- Tabbing is off here (I think you accidentally added tab after `this` instead of before)
- I'd recommend extending `PureComponent` than applying the the mixin this way

```js
import React, { Component } from 'react';

class TooltipInstance extends PureComponent {
	// ...
```

(`React.PureComponent` was introduced in [15.3.0](https://github.com/facebook/react/blob/master/CHANGELOG.md#1530-july-29-2016))